ocp_resources/resource.py

```diff
@@ -299,6 +299,7 @@ class ApiGroup:
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VELERO_IO = "velero.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        OCS_OPENSHIFT_IO = "ocs.openshift.io"
```

Comment:
Move to the right place, ordered by alphabetic

Reply:
done